src/tags/index.js

```diff
@@ -200,5 +246,29 @@ module.exports = function(fractal){
             };
         }
     }
-
 };
+
+function split_translation(translated_string, variables) {
```

Comment:
Is there a reason why this function is declared outside of the exports?

Reply:
Since it is not needed anywhere else? Why would i export something that is not needed anywhere else?

Reply:
@sun did you read my reply?